src/Microsoft.Graph/Models/Extensions/ChangeNotificationCollection.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.IdentityModel.Protocols;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Graph
+{
+    public partial class ChangeNotificationCollection
+    {
+        /// <summary>
+        /// Validates tokens attached with the notification collection. If the result is false, the notification collection should be discarded.
+        /// </summary>
+        /// <param name="tenantIds">List of tenant ids that notifications might be originating from.</param>
+        /// <param name="appIds">List of application id (clientid) that subscriptions have been created from.</param>
+        /// <param name="wellKnownUri">Well known URL to get the signing certificates for the tokens. If you are not using the public cloud you need to pass the value corresponding to your national deployment.</param>
+        /// <param name="issuerPrefix">Issuer prefix for the "aud" claim in the tokens. If you are not using the public cloud you need to pass the value corresponding to your national deployment.</param>
+        /// <returns>Are tokens valid or not.</returns>
+        public async Task<bool> AreTokensValid(IEnumerable<Guid> tenantIds, IEnumerable<Guid> appIds, 
+            string wellKnownUri = "https://login.microsoftonline.com/common/.well-known/openid-configuration",
```

Comment:
I thought leaving the defaults here make it easier for SDK users to know what the values are. Let me know if there's a enum or set of constants I should be leveraging instead.

Reply:
Keeping the constants close to the place where they are used when they are not used widely seems logical to me.  Putting them in a constants file for the sake of conformity doesn't excite me.